build.gradle

```diff
@@ -18,7 +18,7 @@ allprojects {
     }
 
     group = 'com.xfinity'
-    version = '1.0.1'
+    version = '1.1.0'
```

Comment:
Can you remove this please?  Versioning should be in its own PR, and I'd prefer for the project admins to handle it.  Thanks